packages/utility/agent.ts

```diff
@@ -118,7 +118,10 @@ const Opportunities = {
   list: () => requests.get<Opportunity[]>("opportunities"),
 };
 
-const Ideas = {};
+const Ideas = {
+  get: async (params?: URLSearchParams) => 
+    requests.get<Idea[]>("idea-cards", params).then(res => res)
```

Comment:
.then here is redundant

Reply:
Thanks. Just updated to remove this and support more pages within IdeaSpace on v4